blockchain/chain_sync/src/sync.rs

```diff
@@ -46,6 +47,23 @@ enum ChainSyncState {
     Follow,
 }
 
+/// Struct that defines syncing configuration options
+#[derive(Debug, Deserialize, Clone)]
+pub struct SyncConfig {
+    /// Request window length for tipsets during chain exchange
+    pub req_window: i64,
+    /// Number of tasks spawned for sync workers
+    pub worker_tasks: usize,
+}
+
+impl Default for SyncConfig {
+    fn default() -> Self {
+        Self {
+            req_window: 200,
+            worker_tasks: 1,
```

Comment:
can you please add the TODO that existed to revisit this value, as this certainly shouldn't stay once side effects fixed?